### ISSUE TITLE
Need to have support for udev in k3os

### ISSUE THREAD

**Issue by chtardif** [rancher/k3os#151]
Primary use case: OpenEBS/cStor not working without proper udev support in underlying OS

**Comment by jstebbins** [comment]
FYI, Rook/EdgeFS also needs this to identify devices in a way that is persistent across reboots.

**Comment by Alxandr** [comment]
Is this hard to add? Or is it just copying another package from upstream alpine (which as far as I've understood k3os is based on)?

**Comment by liyimeng** [comment]
This is very easy to add, just install it in the base images.
I don’t suggest to have it by default since it increases the initrd size. If you need it, just add it and build from source. Easy job .

**Comment by dweomer** [comment]
@jstebbins and/or @chtardif  If you have a test setup or some guidance (that hopefully doesn't require hours of reading) on how I can assert that your udev use-case is satisfied, I would very much welcome the input. Have any links to share?

**Comment by jstebbins** [comment]
Unfortunately, my use case is rather difficult to set up.

I had previously added udev myself, but I enabled the service by adding this to Dockerfile at the end
```
+RUN rc-update add udev sysinit
+RUN rc-update add udev-trigger sysinit
+RUN rc-update add udev-settle sysinit
```

It looks like you enabled the udev service in a different way, but looks like you're missing udev-trigger and udev-settle?  TBH, it's been so long I can't remember exactly why I added those last 2.  Might just have copied an example I saw elsewhere.
